src/OrchardCore.Themes/TheBlogTheme/TheBlogTheme.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
@jtkech I know that you won't like that one.

Reply:
No particular problem with this line ;)

In fact the question is do we really need a razor view?

If so we should use the razor sdk.

Reply:
I don't know, can we override this with a Liquid view? It requires using a ViewModel that comes from the OrchardCore.Lucene module. We might want to think about an alternative if we want this to work without overriding that module view.